packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -38,6 +38,8 @@ type SelectOpenerProps = AriaProps & {
      * of this component. A placeholder has more faded text colors and styles.
      */
     isPlaceholder: boolean;
+
+    placeholder?: string;
```

Comment:
nit: What do you think about renaming the `placeholder` prop to `ariaLabel` so it is named after what it's used for? We also have the `isPlaceholder` prop so I was wondering how they were related (they're not 😅). Or adding prop jsdocs could help with understanding!

Reply:
Good idea! I see the other `isPlaceholder` prop is for styling. I mentioned in my other comment that we have to be careful with `aria-label` to ensure the correct label is computed, so maybe this other prop can play a role in that logic.